en/components/combo.md

```diff
@@ -106,6 +106,83 @@ When igxCombo is opened, allow custom values are enabled and add item button is
+
+### Code snippets
+```scss
+// in component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+$my-primary-color: #ffffff;
+$my-secondary-color: #7344df;
+
+$my-color-palette: igx-palette(
+    $primary: $my-primary-color,
+    $secondary: $my-secondary-color
+);
+
+$my-light-purple: igx-color($my-color-palette, "secondary", 100);
+$my-medium-purple: igx-color($my-color-palette, "secondary", 300);
+$my-medium-plus-purple: igx-color($my-color-palette, "secondary", 400);
+$my-dark-purple: igx-color($my-color-palette, "secondary", 600);
+
+$custom-drop-down-theme: igx-drop-down-theme(
+    $palette: $my-color-palette,
+    $background-color: $my-light-purple,
+    $item-text-color: $my-primary-color,
+    $header-text-color: $my-dark-purple,
+
+    $selected-item-background: $my-medium-plus-purple,
+    $selected-item-text-color: $my-primary-color,
+    $selected-hover-item-background: $my-medium-plus-purple,
+    $selected-hover-item-text-color: $my-primary-color,
+    $selected-focus-item-background: $my-medium-plus-purple,
+    $selected-focus-item-text-color: $my-primary-color,
+
+    $focused-item-background: $my-medium-purple,
+    $focused-item-text-color: $my-primary-color,
+
+    $hover-item-background: $my-primary-color,
+    $hover-item-text-color: $my-dark-purple
+);
+
+// igx-combo-theme exposes several parameters on top of the igx-drop-down-theme.
+// change $search-separator-border-color to one matching better our purple theme
+$custom-combo-theme: igx-combo-theme(
+    $search-separator-border-color: $my-dark-purple
+);
+```
+
+### Applying
+All that's left is to properly scope our newly created themes. Here we will assume you want to style a particular `IgxCombo` so the other components of this type in your application will not be affected by the custom themes. For details regarding applying theme globally or scoped, you can refer to [igx-drop-down styling guide](drop_down.md#applying)
+
+```
+// Pass our custom-drop-down-theme and custom-combo-theme to respectively `igx-drop-down` and igx-combo mixins.
+:host {
+   ::ng-deep {
+           @include igx-drop-down($custom-drop-down-theme);
+           @include igx-combo($custom-combo-theme);
+   }
+}
+```
+
+> [!NOTE]
+> The [**IgxCombo**]({environment:angularApiUrl}/classes/igxcombocomponent.html) component uses [IgxOverlay]({environment:angularApiUrl}/interfaces/overlaysettings.html) to hold and display the `igx-combo-items` list container. As the overlay is not displayed under the `igx-combo` but by default attached to the document body directly, if the styles are scoped they will not be applied to the `igx-combo` container. This requires the use of `outlet` to manually set the `overlay`to be attached to an element inside the custom theme defined scoped area. For details check: [`IgxOverlay styling guide`](overlay_styling.md).
```

Comment:
Link for `[IgxOverlay]` should be to `overlay_main.md`.
Also, while technically correct, the point we should emphasize is that the overlay is not displayed under our **custom component's host** (not "under the `igx-combo`"). Because, when we provide an `outlet` for the service, it's still not under `igx-combo`, but it's under our component's `host`, so the styles properly apply.
Consider the note for the drop-down section.